### DISCUSSION TITLE
issue and PR status shows "from self and collaborators" for organization

### DISCUSSION THREAD

**Discussion by njzjz** [lowlighter/metrics#800] [Q&A] [answered]
It's unclear for an organization what "self" is. 
https://github.com/lowlighter/metrics/blob/0a9df6270b18988c562856c1950c624117dd6136/source/templates/classic/partials/followup.ejs#L69

**Comment by lowlighter** [comment, marked as answer]
Yeah we should change it for `From members and collaborators` when it's an organization

**Reply by lowlighter** [reply]
Change will be shipped in next release
